### PR TITLE
Chore/1723 add GitHub link

### LINKED ISSUE #1723: Make it really easy to figure out where to enter an issue against PatternFly
We hear frequently that people don't know where to open issues when they are using PatternFly.  It would be great to have something really noticeable on patternfly.org, a clear message at the top of our github org and clear directions on github.com/patternfly/patternfly as the main landing place for new issues.

### PR DESCRIPTION
**Pull request by evwilkin**
Closes #1723 

This PR:
- changes the "Contribute" CTA on the homepage to "Github" and links to github.com/patternfly
- adds a Github icon to the masthead which also links to the PF Github page

### CODE REVIEW

**Review by mcarrano — COMMENTED** [review]
I see what you mean @evwilkin . Is there a way to set a min-width on the button to force it to be the same as the Get Started button? You can also try adding the GitHub icon in front of the label that will make it wider. @mceledonia any other idea?

**Comment by evwilkin** [comment]
@mcarrano I like your suggestion of adding the GH logo to the CTA, I've made that update and will follow up with @mceledonia when he's back online.

**Comment by mcarrano** [comment]
I like that better, too!

**Comment by evwilkin** [comment]
@mceledonia was hoping to get your input on this mockup with the Github CTA on the homepage and the Github icon linked at the top of the page - we had discussed this previously but hadn't seen anything put together yet:

- https://patternfly-org-pr-2243-v4.surge.sh/v4

File: packages/theme-patternfly-org/layouts/sideNavLayout/sideNavLayout.js [hunk 1]
```diff
@@ -47,6 +50,17 @@ const HeaderTools = ({
...
+      <PageHeaderToolsItem>
+        <Text
+          component={TextVariants.a}
+          href="//github.com/patternfly"
+          target="top"
+          aria-label="Link to PatternFly GitHub page"
+          className="ws-github-pageheader pf-u-mr-md"
+        >
+          <GithubIcon />
+        </Text>
```
Comment (redallen) [inline packages/theme-patternfly-org/layouts/sideNavLayout/sideNavLayout.js:62]: This doesn't take up the full width/height and it's difficult to click on. Maybe go with a button instead like the search icon?
image

**Review by redallen — CHANGES_REQUESTED** [review]

**Review by redallen — CHANGES_REQUESTED** [review]
Can we move this to the right of the release dropdown? It feels a little weird that the search box moves the button around:
image

**Comment by evwilkin** [comment]
@redallen I see what you mean about the icon being pushed around by Search - I've swapped it & the search icon/box but still think the version switcher (and the dropdown arrow beside it) make sense to keep as-is.

**Review by redallen — APPROVED** [review]
That's fine, LGTM!

**Review by mcarrano — APPROVED** [review]
This looks great!

**Pull request merged**